.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 coverage
+.vscode/
```

Comment:
Let's remove these.

Reply:
Remove them from the .gitignore? I don't think we want to add my personal VSCode preferences to the repo. I also don't want to accidentally pull down someone else's prefs that blow away my own.